src/scala/com/github/johnynek/bazel_deps/AetherResolver.scala

```diff
@@ -194,7 +264,13 @@ class AetherResolver(servers: List[MavenServer], resolverCachePath: Path) extend
         case "system" => false // these should not be in maven, and should be handled by replacements
         case "import" =>
           // This means pull all the dependencies from a pom we are pointing to
-          sys.error("unsupported")
+          importScopeHandling match {
+            case ImportScopeHandling.Warn =>
+              logger.warn (s"unsupported import-scope dependency: $d")
+              false
+            case ImportScopeHandling.Error =>
+              sys.error (s"unsupported import-scope dependency: $d")
```

Comment:
do you have any idea how much work it is to support this? I have not looked in detail, but I think it also coming up in #220